test/javascripts/acceptance/composer-test.js.es6

```diff
@@ -659,6 +659,27 @@ QUnit.test("Loading draft also replaces the recipients", async assert => {
   assert.equal(find(".users-input .item:eq(0)").text(), "codinghorror");
 });
 
+QUnit.test(
+  "Deleting the text content of the first post in a private message",
+  async assert => {
+    Discourse.SiteSettings.allow_uncategorized_topics = false;
+
+    await visit("/t/34");
+
+    await click("#post_1 .d-icon-ellipsis-h");
+
+    await click("#post_1 .d-icon-pencil-alt");
+
+    await fillIn(".d-editor-input", "");
+
+    assert.equal(
+      find(".d-editor-container textarea").attr("placeholder"),
```

Comment:
Is this actually testing anything? I don't think we ever change the value of the `placeholder' attribute of the composer `textarea` element.

Reply:
Of course it is testing something. Wenn you comment out the fix the placeholder will be "You must select a category before typing here". I mosty write test driven.

Reply:
I see, can this call `I18n.t` instead of copy-pasting the string? If we ever change that string, this test will fail for no good reason.